cf/commands/application/ssh_test.go

```diff
@@ -281,7 +281,7 @@ var _ = Describe("SSH command", func() {
 
 			Context("Error when connecting", func() {
 				It("notifies users", func() {
-					fakeSecureShell.ConnectReturns(errors.New("dial errorrr"))
+					fakeSecureShell.ConnectReturns(errors.New("dial error"))
```

Comment:
This repository is one of the cuter repositories I've seen in a while. For things which were clearly slang, I've left them alone. For things that were borderline, I sometimes changed them. This is borderline.